### PR TITLE
Fix a broken MD link in callout

### PR DESCRIPTION
**Pull request by leemyongpakvn**
## What?

MarkDown link broken in callout div. Use pure HTML link to solve it.

## Testing Instructions

Use MD editor Preview or Open https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/writing-your-first-block-type/ to see how MD link broken.

## Screenshots or screencast 
MD_link_broken_in_Callout

### CODE REVIEW

File: docs/how-to-guides/block-tutorial/writing-your-first-block-type.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This guide takes you through creating a basic block to display a message in a po
 There are two main types of blocks: static and dynamic, this guide focuses on static blocks. A static block is used to insert HTML content into the post and save it with the post. A dynamic block builds the content on the fly when rendered on the front end, see the [dynamic blocks guide](/docs/how-to-guides/block-tutorial/creating-dynamic-blocks.md).
 
 <div class="callout callout-alert">
-This guide focuses on just the block, see the [Create a Block tutorial](/docs/getting-started/create-block/README.md) for a complete setup.
+This guide focuses on just the block, see the <a href="/docs/getting-started/create-block">Create a Block tutorial</a> for a complete setup.
```
Comment (t-hamano) [inline docs/how-to-guides/block-tutorial/writing-your-first-block-type.md]:
```suggestion
This guide focuses on just the block, see the <a href="https://developer.wordpress.org/block-editor/getting-started/create-block/">Create a Block tutorial</a> for a complete setup.
```

If it is a relative path, when this change is reflected in the handbook, the following URL that does not exist will be accessed.

```
https://developer.wordpress.org/docs/getting-started/create-block
```

It's better to use the full URL.

File: packages/components/README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ _If you're using [`Popover`](/packages/components/src/popover/README.md) or [`To
 
 By default, the `Popover` component will render within an extra element appended to the body of the document.
 
-If you want to precisely contol where the popovers render, you will need to use the `Popover.Slot` component.
+If you want to precisely control where the popovers render, you will need to use the `Popover.Slot` component.
```
Comment (t-hamano) [inline packages/components/README.md]: This is definitely a typo, but since it is outside the scope of this PR and is a change related to a specific package, it would be better to submit it as a separate PR.

**Review by t-hamano — COMMENTED** [review]
Thanks for the PR!

I have encountered a similar problem with #50437 etc 😄 I've left some comments, so I'd be happy if you could check them out.

File: packages/components/README.md (hunk 2 above)
Comment (leemyongpakvn) [reply packages/components/README.md]: Absolute URL is applied. Fix a collection of typos PR is coming ;)

**Review by t-hamano — APPROVED** [review]
LGTM! Thank you for updating 👍

**Comment by jeffpaul** [comment]
@leemyongpakvn note that we're compiling credits for the WordPress 6.5 release and would like to confirm your WordPress.org username so that you're properly included in the credits. Mind responding / linking your GH.com and WP.org accounts as noted by the bot comment above?
